cpp/pybind/geometry/boundingvolume.cpp

```diff
@@ -83,6 +83,27 @@ The returned bounding box is an approximation to the minimal bounding box.
      open3d.geometry.OrientedBoundingBox: The oriented bounding box. The
      bounding box is oriented such that the axes are ordered with respect to
      the principal components.
+)doc")
+            .def_static("create_minimal_from_points",
```

Comment:
Can we name `create_from_points_minimal` instead so that it matches the C++ function?

Reply:
Done. Is there any timeline regarding the 0.19 release?

Reply:
As soon as we can get the Python 3.12 support PR merged. There are still some test failures blocking that.